CMakeLists.txt

```diff
@@ -47,14 +47,13 @@ if(APPLE)
 endif()
 
 # Options
-option(BUILD_TORCHTEXT_PYTHON_EXTENSION "Build Python extension" OFF)
```

Comment:
The only time this option should be enabled is when this is being called from Python, and in that case, this option is provided, so I recommend to keep this OFF.